client/extensions/woocommerce/app/order/order-details/table.js

```diff
@@ -93,28 +93,28 @@ class OrderDetailsTable extends Component {
 	renderTableHeader = () => {
 		const { translate } = this.props;
 		return (
-			<TableRow className="order__detail-header">
-				<TableItem isHeader className="order__detail-item-product">{ translate( 'Product' ) }</TableItem>
-				<TableItem isHeader className="order__detail-item-cost">{ translate( 'Cost' ) }</TableItem>
-				<TableItem isHeader className="order__detail-item-quantity">{ translate( 'Quantity' ) }</TableItem>
-				<TableItem isHeader className="order__detail-item-tax">{ translate( 'Tax' ) }</TableItem>
-				<TableItem isHeader className="order__detail-item-total">{ translate( 'Total' ) }</TableItem>
+			<TableRow className="order-details__header">
+				<TableItem isHeader className="order-details__item-product">{ translate( 'Product' ) }</TableItem>
+				<TableItem isHeader className="order-details__item-cost">{ translate( 'Cost' ) }</TableItem>
+				<TableItem isHeader className="order-details__item-quantity">{ translate( 'Quantity' ) }</TableItem>
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Quantity:' )` *ES Score: 26*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).